tests/existing-resources/main.tf

```diff
@@ -0,0 +1,27 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.2.0"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+########################################################################################################################
+# App Config
+########################################################################################################################
+
+# Create new App Config instance
+module "app_config" {
+  source                   = "terraform-ibm-modules/app-configuration/ibm"
+  version                  = "1.5.1"
+  region                   = var.region
+  resource_group_id        = module.resource_group.resource_group_id
+  app_config_name          = "${var.prefix}-app-config"
+  app_config_tags          = var.resource_tags
```

Comment:
you might need to use prefix for the `config_aggregator_trusted_profile_name` input to prevent name clashes in our account?